framework/platform/bpl/include/bpl/bpl_err.h

```diff
@@ -26,74 +26,54 @@ extern "C" {
+/* 12 */ ERROR_CODE(BPL_ERR_BH_SLAVE_SOCKET_DISCONNECTED)                                           \
+/* 13 */ ERROR_CODE(BPL_ERR_BH_STOPPED)                                                             \
+/* 14 */ ERROR_CODE(BPL_ERR_SLAVE_CONNECTING_TO_BACKHAUL_MANAGER)                                   \
+/* 15 */ ERROR_CODE(BPL_ERR_SLAVE_MASTER_KEEP_ALIVE_TIMEOUT)                                        \
+/* 16 */ ERROR_CODE(BPL_ERR_SLAVE_INVALID_MASTER_SOCKET)                                            \
+/* 17 */ ERROR_CODE(BPL_ERR_SLAVE_FAILED_CONNECT_TO_PLATFORM_MANAGER)                               \
+/* 18 */ ERROR_CODE(BPL_ERR_SLAVE_PLATFORM_MANAGER_REGISTER_TIMEOUT)                                \
+/* 19 */ ERROR_CODE(BPL_ERR_SLAVE_SLAVE_BACKHAUL_MANAGER_DISCONNECTED)                              \
+/* 20 */ ERROR_CODE(BPL_ERR_SLAVE_STOPPED)                                                          \
+/* 21 */ ERROR_CODE(BPL_ERR_AP_MANAGER_START)                                                       \
+/* 22 */ ERROR_CODE(BPL_ERR_AP_MANAGER_DISCONNECTED)                                                \
+/* 23 */ ERROR_CODE(BPL_ERR_AP_MANAGER_HOSTAP_DISABLED)                                             \
+/* 24 */ ERROR_CODE(BPL_ERR_AP_MANAGER_ATTACH_FAIL)                                                 \
+/* 25 */ ERROR_CODE(BPL_ERR_AP_MANAGER_SUDDEN_DETACH)                                               \
+/* 26 */ ERROR_CODE(BPL_ERR_AP_MANAGER_HAL_DISCONNECTED)                                            \
+/* 27 */ ERROR_CODE(BPL_ERR_AP_MANAGER_CAC_TIMEOUT)                                                 \
+/* 28 */ ERROR_CODE(BPL_ERR_MONITOR_DISCONNECTED)                                                   \
+/* 29 */ ERROR_CODE(BPL_ERR_MONITOR_HOSTAP_DISABLED)                                                \
+/* 30 */ ERROR_CODE(BPL_ERR_MONITOR_ATTACH_FAIL)                                                    \
+/* 31 */ ERROR_CODE(BPL_ERR_MONITOR_SUDDEN_DETACH)                                                  \
+/* 32 */ ERROR_CODE(BPL_ERR_MONITOR_HAL_DISCONNECTED)                                               \
+/* 33 */ ERROR_CODE(BPL_ERR_MONITOR_REPORT_PROCESS_FAIL)                                            \
+/* 34 */ ERROR_CODE(BPL_ERR_CONFIG_PLATFORM_REPORTED_INVALID_CONFIGURATION)                         \
+/* 35 */ ERROR_CODE(BPL_ERR_CONFIG_BACKHAUL_WIRED_INTERFACE_IS_UNSUPPORTED)                         \
+/* 36 */ ERROR_CODE(BPL_ERR_CONFIG_BACKHAUL_WIRELESS_INTERFACE_IS_UNSUPPORTED)                      \
+/* 37 */ ERROR_CODE(BPL_ERR_CONFIG_NO_VALID_BACKHAUL_INTERFACE)                                     \
+/* 38 */ ERROR_CODE(BPL_ERR_WATCHDOG_PROCESS_STUCK)                                                 \
+/* 39 */ ERROR_CODE(BPL_ERR_WATCHDOG_PROCESS_ZOMBIE)                                                \
+/* 40 */ ERROR_CODE(BPL_ERR_UNKNOWN)                                                                \
```

Comment:
```
/* 40 */ ERROR_CODE(BPL_ERR_UNKNOWN)                                                                \
/* 41 */ ERROR_CODE(BPL_ERR_OPERATION_NOT_SUPPORTED)  
```
Have to be 1 and 2 respectively.

Reply:
why? to make it 1 and 2 I need to change all other numbers 😱

Reply:
So do it or remove the numbers :/

Reply:
Add a fixup. I kept the numbering since it is helpful when debugging since the error notification prints a number.